google/cloud/storage/internal/async/connection_tracing.cc

```diff
@@ -15,6 +15,8 @@
 #include "google/cloud/storage/internal/async/connection_tracing.h"
 #include "google/cloud/storage/internal/async/reader_connection_tracing.h"
 #include "google/cloud/internal/opentelemetry.h"
+#include "google/cloud/internal/version.h"
```

Comment:
This cannot be needed.  `async/connection_tracing.h` must include one of the `*/version.h` files.

Reply:
https://screenshot.googleplex.com/587x5ktYjxJinhK

🤷 

Should I remove it?

Reply:
I suspect we need to change the several `*/version.h` files to tell IWYU "hey, I intentionally export the symbols from this other header, don't bother people about them".  I just don't know how to do such a thing.

Reply:
I don't either at the moment... 🤷